hasura/metadata/databases/default/tables/public_circles.yaml

```diff
@@ -137,15 +137,16 @@ select_permissions:
     - vouching
     - vouching_text
     filter:
-      organization:
-        circles:
-          users:
-            _and:
-            - profile:
-                id:
-                  _eq: X-Hasura-User-Id
-            - deleted_at:
-                _is_null: true
+      _and:
+      - deleted_at:
+          _is_null: true
+      - users:
+          _and:
+          - profile:
+              id:
+                _eq: X-Hasura-User-Id
+          - deleted_at:
+              _is_null: true
```

Comment:
This actually breaks functionality for user-level members and fails to return circles a user is not a member in but is within the org. These entries from our test seed are missing in the UI: 
![image](https://user-images.githubusercontent.com/17910833/185457593-7bea160c-7e93-4bdf-bb3a-db8b228d957f.png)

I recommend this instead:

```suggestion
      - organization:
          circles:
            users:
              _and:
              - profile:
                  id:
                    _eq: X-Hasura-User-Id
              - deleted_at:
                  _is_null: true
      - deleted_at:
          _is_null: true

```